vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -1023,11 +1047,53 @@ void WindowsTextInputComponentView::DrawText() noexcept {
         static_cast<LONG>(offset.x) + static_cast<LONG>(m_imgWidth),
         static_cast<LONG>(offset.y) + static_cast<LONG>(m_imgHeight)};
 
-    winrt::check_hresult(m_textServices->OnTxInPlaceActivate(&rcClient));
-
-    // TODO keep track of proper invalid rect
-    auto hrDraw = m_textServices->TxDrawD2D(d2dDeviceContext.get(), &rc, nullptr, TXTVIEW_ACTIVE);
-    winrt::check_hresult(hrDraw);
+    // draw text ontop of richedit
+    if (!m_placeholderText.empty() && m_firstTextUpdate <= 2) {
+      m_firstTextUpdate++; // TODO: Figure out why DrawText is/needs to be called twice to actually draw
+      winrt::com_ptr<ID2D1SolidColorBrush> brush;
+      winrt::check_hresult(
+          d2dDeviceContext->CreateSolidColorBrush(D2D1::ColorF(D2D1::ColorF::Green, 1.0f), brush.put()));
+
+      facebook::react::LayoutConstraints constraints;
+      constraints.maximumSize.width = static_cast<FLOAT>(offset.x) + static_cast<FLOAT>(m_imgWidth);
+      constraints.maximumSize.height = static_cast<FLOAT>(offset.y) + static_cast<FLOAT>(m_imgHeight);
+
+      // Create a fragment with text attributes
+      facebook::react::AttributedString attributedString;
+      facebook::react::AttributedString::Fragment fragment1;
+      facebook::react::TextAttributes textAttributes;
+      // facebook::react::ShadowView parentShadowView;
+      m_textLayout = nullptr;
+
+      textAttributes.fontSize = 16.0f; // TODO: should be m_props->textAttributes.fontSize but breaks rntester
+      fragment1.string = m_placeholderText;
+      fragment1.textAttributes = textAttributes;
+      // fragment1.parentShadowView = parentShadowView; // do I need to find the parent shadow view maybe??
+      attributedString.appendFragment(fragment1);
+
+      m_attributedStringBox = facebook::react::AttributedStringBox(attributedString);
+      facebook::react::TextLayoutManager::GetTextLayout(m_attributedStringBox, {} /*TODO*/, constraints, m_textLayout);
+
+      // draw text
+      d2dDeviceContext->DrawTextLayout(
+          D2D1::Point2F(static_cast<FLOAT>(4.0f), static_cast<FLOAT>(4.0f)),
```

Comment:
I think this should be:
 D2D1::Point2F(
            static_cast<FLOAT>((offset.x + m_layoutMetrics.contentInsets.left) / m_layoutMetrics.pointScaleFactor),
            static_cast<FLOAT>((offset.y + m_layoutMetrics.contentInsets.top) / m_layoutMetrics.pointScaleFactor))